x/ibc/03-connection/types/msgs.go

```diff
@@ -99,6 +105,9 @@ func (msg MsgConnectionOpenTry) ValidateBasic() error {
 	if err := host.ClientIdentifierValidator(msg.ClientId); err != nil {
 		return sdkerrors.Wrap(err, "invalid client ID")
 	}
+	if msg.ClientState == nil {
+		return sdkerrors.Wrap(clienttypes.ErrInvalidClient, "counterparty client is nil")
```

Comment:
missing case

Reply:
I believe this is tested already here: https://github.com/cosmos/cosmos-sdk/pull/7057/files#diff-99ed87279f42549ae584c48e60048e50R114

Reply:
Ahh my bad, these were failing earlier, fixed now.